src/stable/test/jest_link.ts

```diff
@@ -1,16 +1,26 @@
 import { execSync } from 'child_process';
-import { join, resolve } from 'path';
+import { join } from 'path';
+
+import { AZLE_ROOT } from '#build/utils/global_paths';
```

Comment:
Is this a problem? Should we be getting the azle root another way instead of import.meta?

Reply:
The old way I'm guessing is going to work just fine